sources/packages/web/src/components/aest/institution/modals/AssessOfferingModal.vue

```diff
@@ -0,0 +1,85 @@
+<template>
```

Comment:
It seems that are creating more and more modals that are doing exactly the same: loading a form and Approve/Decline. Can we create a component for it and just reuse it?
Create the tech-debit for it: https://app.zenhub.com/workspaces/student-information-management-system-5fce9df5aa1b45000e937014/issues/bcgov/sims/1207

Reply:
Yes makes sense. And when we build that component probably we can built it with vuetify 3 when it is complete.